job-server/src/main/scala/spark/jobserver/AkkaClusterSupervisorActor.scala

```diff
@@ -215,10 +113,12 @@ class AkkaClusterSupervisorActor(daoActor: ActorRef) extends InstrumentedActor {
     }
 
     val pb = Process(cmdString)
-    val pio = new ProcessIO(_ => (),
-                        stdout => scala.io.Source.fromInputStream(stdout)
-                          .getLines.foreach(println),
-                        stderr => scala.io.Source.fromInputStream(stderr).getLines().foreach(println))
+    val pio = new ProcessIO(
+      (_) => (),
```

Comment:
yeah... I debated that one. I looked around and saw examples of both in a few places, so wasn't sure it was super standard across the project, and moving the first anonymous function onto its own line, it seemed to read a lot better to me. I can keep the spacing and remove the `()` parens, but I do think long term parens around the single arg functions reads quite a bit better (I mostly hold that from writing a lot ES6 where that is becoming the norm), but its definitely not too important